core/package.json

```diff
@@ -20,10 +20,9 @@
     "nuxeo": "^3.2.1"
   },
   "devDependencies": {
-    "@open-wc/testing-helpers": "^0.8.3",
+    "@nuxeo/nuxeo-testing-helpers": "3.0.0-SNAPSHOT",
```

Comment:
WDYT of going with just `@nuxeo/testing-helpers`? I know we have the `nuxeo-` prefix on elements' packages but custom elements do need a prefix...